.changeset/wicked-wombats-look.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: `font-weight-notation` messages
```

Comment:
[ask]

- This PR removes the validation path, but is it a patch change? Or, a minor change?
- "Fixed: `font-weight-notation` messages" may be inaccurate. Any better description?

Reply:
I think "Fixed: font-weight-notation messages" is fine.